src/MagnumPlugins/UfbxImporter/Test/UfbxImporterTest.cpp

```diff
@@ -1180,52 +1172,49 @@ void UfbxImporterTest::geometricTransformNoGeometric() {
+         {},                                   /* Box002: identity */
+    }), TestSuite::Compare::Container);
+    CORRADE_COMPARE_AS(geometryScalingsFloat, Containers::arrayView<Vector3>({
+         {1.0f, 1.0f, 2.0f},
+         {1.0f, 1.0f, 1.0f},
+    }), TestSuite::Compare::Container);
 }
 
-void UfbxImporterTest::geometricTransformNoGeometricPreserveRoot() {
+void UfbxImporterTest::geometricTransformInvalidHandling() {
     Containers::Pointer<AbstractImporter> importer = _manager.instantiate("UfbxImporter");
-    importer->configuration().setValue("geometricTransformNodes", false);
-    importer->configuration().setValue("preserveRootNode", true);
-    CORRADE_VERIFY(importer->openFile(Utility::Path::join(UFBXIMPORTER_TEST_DIR, "geometric-transform.fbx")));
-
-    /* Two real objects and root */
-    CORRADE_COMPARE(importer->objectCount(), 3);
-    CORRADE_COMPARE(importer->objectName(0), "");
-    CORRADE_COMPARE(importer->objectName(1), "Box001");
-    CORRADE_COMPARE(importer->objectName(2), "Box002");
+    importer->configuration().setValue("geometryTransformHandling", "invalidHandling");
 
-    Containers::Optional<SceneData> scene = importer->scene(0);
-    CORRADE_VERIFY(scene);
-
-    CORRADE_VERIFY(scene->hasField(SceneField::Parent));
-    CORRADE_VERIFY(scene->hasField(SceneField::Translation));
-    CORRADE_VERIFY(scene->hasField(SceneField::Rotation));
-    CORRADE_VERIFY(scene->hasField(SceneField::Scaling));
-
-    CORRADE_COMPARE_AS(scene->field<Int>(SceneField::Parent), Containers::arrayView<Int>({
-        -1, 0, 1,
-    }), TestSuite::Compare::Container);
-
-    /* The meshes should be parented under their geometric transform helpers */
-    CORRADE_COMPARE_AS(scene->meshesMaterialsAsArray(), (Containers::arrayView<Containers::Pair<UnsignedInt, Containers::Pair<UnsignedInt, Int>>>({
-        {1, {0, -1}},
-        {2, {1, -1}},
-    })), TestSuite::Compare::Container);
+    std::ostringstream out;
+    Error redirectError{&out};
 
-    const Containers::Pair<UnsignedInt, Containers::Triple<Vector3,Quaternion,Vector3>> referenceTrs[] = {
-        /* (root) */
-        {0, {{0.0f, 0.0f, 0.0f}, {}, {1.0f, 1.0f, 1.0f}}},
-        /* Box001 */
-        {1, {{0.0f, -10.0f, 0.0f},
-             {{0.0f, 0.0f, -0.707107f}, 0.707107f}, /* Euler (0, 0, -90) */
-             {1.0f, 2.0f, 1.0f}}},
-        /* Box002 */
-        {2, {{0.0f, 0.0f, 20.0f},
-             {{0.0f, 0.0f, -1.0f}, 0.0f}, /* Euler (0, 0, -180) */
-             {1.0f, 0.5f, 1.0f}}},
-    };
-    CORRADE_COMPARE_AS(scene->translationsRotationsScalings3DAsArray(),
-        Containers::arrayView(referenceTrs), TestSuite::Compare::Container);
+    CORRADE_VERIFY(importer->openFile(Utility::Path::join(UFBXIMPORTER_TEST_DIR, "geometric-transform.fbx")));
+    CORRADE_COMPARE(out.str(), "Trade::UfbxImporter::openFile(): Unsupported geometryTransformHandling configuration: invalidHandling\n");
```

Comment:
I left this intentionally vague for now so I don't waste time implementing behavior that is not wanted. What should we do on invalid configuration? Just hard fail out right or fall back to default, currently it falls back to ufbx's default (preserve) which is different from the Magnum default (helperNodes), which is pretty confusing..

Reply:
Failing sounds better to me, other plugins do the same in the presence of invalid config values. Because otherwise, even if there was a typo (such as using `helper_nodes` instead of `helperNodes`), the warning could get buried in some verbose log output and people would be confused that it didn't do the right thing.